lib/linguist/languages.yml

```
@@ -940,6 +940,75 @@ Charity:
   tm_scope: none
   ace_mode: text
   language_id: 56
+Checksums:
+  type: data
+  tm_scope: text.checksums
+  aliases:
+  - checksum
+  - hash
+  - hashes
+  - sum
+  - sums
+  filenames:
+  - CRC32
+  - CRC32SUM
+  - CRC32SUMS
+  - CRCSUM
+  - CRCSUMS
+  - MD2
+  - MD2SUM
+  - MD2SUMS
+  - MD4
+  - MD4SUM
+  - MD4SUMS
+  - MD5
+  - MD5SUM
+  - MD5SUMS
+  - RMD160
+  - RMD160SUM
+  - RMD160SUMS
+  - SHA1
+  - SHA1SUM
+  - SHA1SUMS
+  - SHA2
+  - SHA224
+  - SHA224SUM
+  - SHA224SUMS
+  - SHA256
+  - SHA256SUM
+  - SHA256SUMS
+  - SHA2SUM
+  - SHA2SUMS
+  - SHA3
+  - SHA384
+  - SHA384SUM
+  - SHA384SUMS
+  - SHA3SUM
+  - SHA3SUMS
+  - SHA512
+  - SHA512SUM
+  - SHA512SUMS
+  - checksum
+  - checksums
+  - cksum
```

Not popular enough, though checksums.txt is:

```suggestion
  - checksums.txt
```